app/models/content_configuration.rb

```diff
@@ -12,6 +12,20 @@ class ContentConfiguration < Spree::Preferences::FileConfiguration
   has_attached_file :logo_mobile_svg, default_url: "/assets/ofn-logo-mobile.svg"
 
   # Home page
+  #
+  # I intended to set a default for home_page_alert_html including I18n text.
+  #
+  #      default: "<a href='/register' target='_blank'>
+  #             #{I18n.t('shared.register_call.selling_on_ofn')}&nbsp;
+  #             <strong>#{I18n.t('shared.register_call.register')}
+  #             <i class='ofn-i_054-point-right'></i></strong>
+  #             </a>"
+  #
+  # Unfortunately, this configuration is initialised and cached before locales
+  # are loaded. So I chose to not set a default and use a translatable view instead.
```

Comment:
I dont think we should tell a story in comments. imo the story can go in the commit message or the PR. comments should be straight to the point about the current state of the code. I'd keep only these 2 last lines:
# All the following defaults using I18n don't work.
# https://github.com/openfoodfoundation/openfoodnetwork/issues/3816

Reply:
You are right, I covered that in the issue now. There was a point in my development when that comment made more sense and I didn't want to lose the HTML code to paste. But in the end I kept the view and the default is not need. :+1: